tests/main/views/test_drafts.py

```diff
@@ -48,14 +45,12 @@ def setup(self):
                 .update(dict(status='live'))
             Framework.query.filter_by(slug='g-cloud-7') \
                 .update(dict(status='open'))
-            db.session.commit()
 
-        self.client.put(
-            '/services/%s' % self.service_id,
-            data=json.dumps(
-                {'updated_by': 'joeblogs',
-                 'services': payload}),
-            content_type='application/json')
+            self.setup_dummy_service(
+                service_id=self.service_id,
+                **payload
+            )
+            db.session.commit()
```

Comment:
You're right, Kev. It looks like most of them do do the commit at the end of the method (in [FixtureMixin](https://github.com/alphagov/digitalmarketplace-api/blob/a1f9f4dce3d93d53f11bcce575329ff8e635d525/tests/helpers.py#L43)), so maybe it's worth adding it.